pkg/kong/service.go

```diff
@@ -0,0 +1,79 @@
+package kong
+
+import (
+	"errors"
+	"fmt"
+	"github.com/api7/kong-to-apisix/pkg/apisix"
+	uuid "github.com/satori/go.uuid"
+)
+
+func MigrateService(kongConfig *Config, apisixConfig *apisix.Config) error {
+	kongServices := kongConfig.Services
+	apisixServices := apisixConfig.Services
+
+	for _, kongService := range kongServices {
+		var apisixService apisix.Service
+		apisixService.ID = kongService.ID
+		apisixService.Name = kongService.Name
+		upstreamID := GenerateApisixServiceUpstream(kongService, apisixConfig)
+		apisixService.UpstreamID = upstreamID
+		apisixServices = append(apisixServices, apisixService)
+	}
+
+	apisixConfig.Services = apisixServices
+	return nil
+}
+
+func GenerateApisixServiceUpstream(kongService Service, apisixConfig *apisix.Config) string {
+	var apisixUpstream apisix.Upstream
+	// apisix upstream id
+	apisixUpstream.ID = uuid.NewV4().String()
```

Comment:
Better to encode the kong's service ID as part of upstream ID, so that people know where it comes

Reply:
Kong's upstream has an ID, but the generated upstream is split from the service, so there is no default ID.

Reply:
I mean to include the Kong service's ID as part of the upstream ID